src/material-experimental/mdc-menu/harness/menu-harness-filters.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export type MenuHarnessFilters = {
+  text?: string | RegExp
```

Comment:
should we call it `triggerText` so it's clear which text?

Reply:
Done